clap_man/README.md

```diff
@@ -0,0 +1,20 @@
+<!-- omit in TOC -->
+# clap_man
+
+> **Manpage generation for `clap`**
+
+[![Crates.io](https://img.shields.io/crates/v/clap_man?style=flat-square)](https://crates.io/crates/clap_man)
+[![Crates.io](https://img.shields.io/crates/d/clap_man?style=flat-square)](https://crates.io/crates/clap_man)
+[![License](https://img.shields.io/badge/license-Apache%202.0-blue?style=flat-square)](https://github.com/clap-rs/clap/blob/master/LICENSE-APACHE)
+[![License](https://img.shields.io/badge/license-MIT-blue?style=flat-square)](https://github.com/clap-rs/clap/blob/master/LICENSE-MIT)
+
+Dual-licensed under [Apache 2.0](LICENSE-APACHE) or [MIT](LICENSE-MIT).
+
+1. [About](#about)
+2. [API Reference](https://docs.rs/clap_man)
+3. [Questions & Discussions](https://github.com/clap-rs/clap/discussions)
+4. [CONTRIBUTING](CONTRIBUTING.md)
+5. [Sponsors](../README.md#sponsors)
```

Comment:
If you rebase, you'll see that these links in `clap_generate` have been updated and there is a new section in the `Cargo.toml` file that deals with these

Reply:
Done.